src/libData/AccountStore/services/evm/EvmProcessContext.h

```diff
@@ -73,7 +73,7 @@ struct EvmProcessContext {
                     const zbytes& code, const zbytes& data, const uint64_t& gas,
                     const uint256_t& amount, const uint64_t& blkNum,
                     const TxnExtras& extras, std::string_view context,
-                    bool estimate = false);
+                    bool estimate, bool direct);
```

Comment:
I think this requires some documentation in comments. Just as all the members of this class.

Reply:
Ok, will do.